src/Elastic.Apm/Report/Serialization/LabelsJsonConverter.cs

```diff
@@ -11,7 +11,11 @@ public override void WriteJson(JsonWriter writer, Dictionary<string, string> lab
 			writer.WriteStartObject();
 			foreach (var keyValue in labels)
 			{
-				writer.WritePropertyName(SerializationUtils.TrimToPropertyMaxLength(keyValue.Key));
+				// Labels are trimmed and also de dotted in order to satisfy the Intake API
+				writer.WritePropertyName(SerializationUtils.TrimToPropertyMaxLength(keyValue.Key)
+					.Replace('.', '_')
+					.Replace('*', '_')
```

Comment:
It might be worth updating the docs to mention all the characters that are replaced.

Reply:
I changed the doc - the Go doc had 1 nice sentence covering this. The Java doc does not even talk about this details.